core/src/main/java/com/linecorp/armeria/client/retry/AbstractRetryingClientBuilder.java

```diff
@@ -140,10 +149,9 @@ public String toString() {
     }
 
     final ToStringHelper toStringHelper() {
-        return MoreObjects.toStringHelper(this).omitNullValues()
-                          .add("retryRule", retryRule)
-                          .add("retryRuleWithContent", retryRuleWithContent)
-                          .add("maxTotalAttempts", maxTotalAttempts)
-                          .add("responseTimeoutMillisForEachAttempt", responseTimeoutMillisForEachAttempt);
+        checkState(
+                retryConfig != null,
+                "You are using a RetryConfigMapping, so you cannot get a string representation.");
+        return retryConfig.toStringHelper();
```

Comment:
This will make a debugger see an exception instead of a string. Could we make this fall back to the old version when `retryConfig` is null?

Reply:
Changed it so when a mapping is used instead of retryConfig, the mapping is printed.